### PR TITLE
update 'jubeat ripples'

### PR DESCRIPTION
**Pull request by ikaros14**
Side effects found when skipping mode selection so changed to skip tutorial only

### CODE REVIEW

File: jubeatripples.html [hunk 1]
```diff
@@ -11,18 +11,14 @@
         <script type="text/javascript">
             window.addEventListener("load", function() {
                 new PatchContainer([
-                    new Patcher("jubeat.exe", "2009-12-24 (China ver.)", [
+                    new Patcher("jubeat.exe", "2009-12-24 (China Ver.1)", [
```
Comment (mon) [inline jubeatripples.html:14]: Don't version patches like this - it's still for China ver

Comment (ikaros14) [reply jubeatripples.html:14]: I added it because there are three Chinese versions. Should I really rollback it?

**Comment by mon** [comment]
Whoah what did you do to the diff it's insane lol

**Comment by ikaros14** [comment]
I'm sorry to scare you. It's just a little thing lol

File: jubeatripples.html (hunk 1 above)
Comment (ikaros14) [reply jubeatripples.html:14]: thanks! I'm done with all the modifications

**Comment by mon** [comment]
I'm a little curious which code formatter you installed that has a space after dict key, eg `key: val` vs `key : val`

**Comment by ikaros14** [comment]
Is it common to have no spaces? eg `key: val`

**Comment by mon** [comment]
That is what I see most often in Python and Javascript (see the formatters: [black](https://github.com/psf/black) and [prettier](https://prettier.io/))

**Comment by ikaros14** [comment]
Now I understand. Suddenly, the difference was noticeable, so I didn't use the formatter and unified it. I've done something useless. Should I delete the space again?

**Comment by mon** [comment]
Yes, please delete the space - it makes it very noisy otherwise... Sorry, and thanks again for your patience

**Comment by ikaros14** [comment]
I'm rather sorry I'm stupid

**Pull request merged**